<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageBoxCus</class>
 <widget class="QDialog" name="MessageBoxCus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MessageBoxCus</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>62</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>62</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>logo</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_message">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>title</string>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
